rice/detail/MethodInfo.ipp

```diff
@@ -13,7 +13,8 @@ namespace Rice
     for (size_t i = this->args_.size(); i < argCount; i++)
     {
       Arg arg("arg_" + std::to_string(i));
-      this->args_.emplace_back(arg);
+      // Use addArg instead of args_->emplace_back to avoid std::bad_alloc
```

Comment:
Lets get rid of this comment as it won't make sense after this change is made. Otherwise, we can get this merged.

Reply:
Great, updated